compiler/src/test/scala/org/scalajs/nscplugin/test/JSExportTest.scala

```diff
@@ -100,6 +100,25 @@ class JSExportTest extends DirectTest with TestHelpers {
     """
   }
 
+  @Test
```

Comment:
The problem is right above this line: The `JSExportTopLevel` tests should have been removed.